app/models/manageiq/providers/redhat/inventory_collection_default/infra_manager.rb

```diff
@@ -131,6 +131,23 @@ def root_folders(extra_attributes = {})
       attributes.merge!(extra_attributes)
     end
 
+    def custom_attributes(extra_attributes = {})
```

Comment:
Why not to update [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manager_refresh/inventory_collection_default/infra_manager.rb#L137) instead.

Reply:
I am not sure what else the core part is used for, I will ask @ladas + its faster to merge.